lib/livebook_web/live/home_live.ex

```diff
@@ -110,6 +110,8 @@ defmodule LivebookWeb.HomeLive do
               id="session-list"
               sessions={@sessions}/>
           </div>
+          <div class="sr-only" role="alert" id="live-region">
```

Comment:
Maybe it would make sense to move it to the layout, and then make the helper global like `sr_message(js, message)`, this way we could use it anywhere, wdyt?

Reply:
Good idea! 🐱